internal/backends/sqlite/collection.go

```diff
@@ -287,6 +288,42 @@ func (c *collection) Stats(ctx context.Context, params *backends.CollectionStats
 	}, nil
 }
 
+// ListIndexes implements backends.Collection interface.
+func (c *collection) ListIndexes(ctx context.Context, params *backends.ListIndexesParams) (*backends.ListIndexesResult, error) {
+	db := c.r.DatabaseGetExisting(ctx, c.dbName)
+	if db == nil {
+		return nil, backends.NewError(
+			backends.ErrorCodeDatabaseDoesNotExist,
+			lazyerrors.Errorf("no ns %s.%s", c.dbName, c.name),
+		)
+	}
+
+	coll := c.r.CollectionGet(ctx, c.dbName, c.name)
+	if coll == nil {
+		return nil, backends.NewError(
+			backends.ErrorCodeCollectionDoesNotExist,
+			lazyerrors.Errorf("no ns %s.%s", c.dbName, c.name),
+		)
+	}
+
+	// only one index is supported at the moment - _id
+	// TODO https://github.com/FerretDB/FerretDB/issues/3176
+	return &backends.ListIndexesResult{
+		Indexes: []backends.IndexInfo{
+			{
+				Unique: pointer.ToBool(true),
+				Name:   "_id_",
+				Key: []backends.IndexKeyPair{
+					{
+						Field: "_id",
+						Order: 1,
```

Comment:
So are we using 1/-1 or types.Ascending/Descending?

Reply:
fixed